presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -311,7 +311,12 @@ void PrestoServer::run() {
           std::shared_ptr<velox::exec::ExchangeQueue> queue,
           memory::MemoryPool* pool) {
         return PrestoExchangeSource::create(
-            taskId, destination, queue, pool, exchangeExecutor_);
+            taskId,
+            destination,
+            queue,
+            pool,
+            driverCPUExecutor(),
+            exchangeIoExecutor_.get());
```

Comment:
Shall we call this networkExectuor_ or httpExecutor_? IO could mean disk io as well?

Reply:
Since this is exchange specific let's call it exchangeHttpExecutor_?